frontend/lib/with-apollo-client.tsx

```diff
@@ -97,17 +97,22 @@ const withApolloClient = (App: any) => {
 
       // Extract query data from the Apollo store
       const apolloState = apollo.cache.extract()
+      ;(apollo as any).toJSON = () => {
```

Comment:
What is this doing?

Reply:
// To avoid calling initApollo() twice in the server we send the Apollo Client as a prop
// to the component, otherwise the component would have to call initApollo() again but this
// time without the context, once that happens the following code will make sure we send
// the prop as `null` to the browser

from https://github.com/lfades/next-with-apollo/blob/master/src/withApollo.tsx

Reply:
Isn't really needed in our case, though, was just testing